content/source/docs/enterprise/workspaces/settings.html.md

```diff
@@ -114,6 +114,25 @@ You can use the "Select a VCS connection" buttons and "Repository" field to chan
 
 -> **API:** If you need to change VCS connections for many workspaces at once, consider automating the changes with the [Update a Workspace endpoint](../api/workspaces.html#update-a-workspace). This is most common when moving a VCS server, or when a vendor deprecates an older API version.
 
+### Automatic Run Triggering
+
+For workspaces that **don't** specify a Terraform working directory, TFE assumes that the entire repository is relevant to the workspace. Any change will trigger a run.
+
+For workspaces that **do** specify a Terraform working directory, TFE assumes that only _some_ content in the repository is relevant to the workspace. Only changes that affect the relevant content will trigger a run. By default, only the working directory is considered relevant.
+
+You can adjust this behavior in two ways:
+
+- **Add more trigger directories.** TFE will queue runs for changes in any of the specified trigger directories (including the working directory).
+
+    For example, if you use a top-level `modules` directory to share Terraform code across multiple configurations, changes to the shared modules are relevant to every workspace that uses that repo. You can add `modules` as a trigger directory for each workspace to make sure they notice any changes to shared code.
+- **Mark the entire repository as relevant.** If you set the "Automatic Run Triggering" setting to "Always Trigger Runs," TFE will assume that anything in the repository might affect the workspace's configuration, and will queue runs for any change.
+
+    This can be useful for repos that don't have multiple configurations but require a working directory for some other reason. It's usually not what you want for true monorepos, since it queues unnecessary runs and slows down your ability to provision infrastructure.
```

Comment:
Do people reading this know what a `monorepo` is? Should we link to the [repo structure](https://www.terraform.io/docs/enterprise/workspaces/repo-structure.html#directories) section (and possibly link back here?

Reply:
I think monorepo is probably well-understood, at least with the extra context, and I can't find a good way to link to the repo structure page from here. I did add a link from the repo structure page back to this one, though.